### ISSUE TITLE
Cannot deploy since flyctl v0.1.149: cannot apply host to transport: *otelhttp.Transport

### ISSUE THREAD

**Issue by dronda-t** [superfly/flyctl#3274]
### Describe the bug
I deploy fly apps using a local docker image. I do `flyctl deploy --local-only`. I also use colima for docker.

After upgrading to flyctl 0.1.149, the deploy fails saying no docker daemon available. I have tried upgrading to 0.2.0, where I got the same error, and downgrading to 0.1.148 where I am able to deploy successfully.

* Operating system: MacOS Sonoma 14.3.1
* `fly version`: 0.1.149

### Command output:

```text
$ fly deploy --local-only                                                                                                                                                                       (02-10 10:54)
==> Verifying app config
Validating /Users/todd/projects/personal/photo-processor/fly.toml
✓ Configuration is valid
--> Verified app config
==> Building image
WARN Error connecting to local docker daemon:cannot apply host to transport: *otelhttp.Transport
WARN failed to create build in graphql: input:3: createBuild Could not find App


Error: failed to fetch an image or build from source: no docker daemon available
```

**Comment by kzys** [comment]
Let me take a look. Can you try `fly docker`? What is the version of colima?

**Comment by dronda-t** [comment]
Thanks @kzys.

There doesn't seem to be a `fly docker` command. It just says unknown command.

In case you meant doctor, this is the output on (0.2.0):

```
$ fly doctor
Testing authentication token... PASSED
Testing flyctl agent... PASSED
Testing local Docker instance... Nope
Pinging WireGuard gateway (give us a sec)... PASSED
Testing WireGuard DNS... PASSED
Testing WireGuard Flaps... PASSED

App specific checks for photo-processor:
Checking that app has ip addresses allocated... PASSED
No public ipv4 or ipv6 ip addresses allocated to app photo-processor

Build checks for photo-processor:
Checking docker context size (this may take little bit)... PASSED (54 kB)
```

Colima version:
```
$ colima --version
colima version 0.6.8
```

**Comment by kzys** [comment]
@dronda-t Yes. Sorry. `fly doctor` was right.

We've just released 0.2.7 which should fix the issue. Let me know if that works!

**Comment by dronda-t** [comment]
Thank you @kzys! I can confirm it's working for me now.
